# Generated by Django 3.1.6 on 2021-02-20 14:07

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('register_api', '0006_auto_20210210_1625'),
    ]

    operations = [
        migrations.CreateModel(
            name='Bug',
            fields=[
                ('id', models.AutoField(primary_key=True, serialize=False)),
                ('bug_name', models.CharField(max_length=100, verbose_name='Nome do bug')),
                ('date', models.DateTimeField(auto_now_add=True, verbose_name='data de criação')),
                ('last_update', models.DateTimeField(auto_now=True, verbose_name='Ultima Atualização')),
                ('description', models.TextField(blank=True, verbose_name='Descrição do Bug')),
                ('status', models.CharField(choices=[('OPEN', 'Em Aberto'), ('EVAL', 'Em Avaliação'), ('SOLV', 'Solucionado')], default='OPEN', max_length=20)),
                ('author', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='+', to=settings.AUTH_USER_MODEL)),
                ('last_modified_by', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='+', to=settings.AUTH_USER_MODEL)),
            ],
            options={
                'verbose_name_plural': 'Bugs',
            },
        ),
        migrations.DeleteModel(
            name='Error',
        ),
    ]
